source/_integrations/roborock.markdown

```diff
@@ -115,7 +115,7 @@ We plan to make the process simpler in the future, but for now, it is a multi-st
 3. Search your logs for 'Got home data' and find the attribute rooms.
 4. Write the rooms down; they have a name and 6 digit ID.
 5. Go to {% my developer_call_service service="vacuum.send_command" title="**Developer Tools** > **Services** > **Vacuum: Send Command**" %}. Select your vacuum as the entity and `get_room_mapping` as the command.
-6. Go back to your logs and look at the response to `get_room_mapping`. This is a list of the 6-digit IDs you saw earlier to 2-digit IDs. In your original list of room names and 6-digit IDs, replace the 6-digit ID with its pairing 2-digit ID.
+6. Go back to your logs and look at the response to `get_room_mapping`. This is a list of the 6-digit IDs you saw earlier to 2-digit IDs (use the first number, for instance `16` in `[16, '14000663', 12]`). In your original list of room names and 6-digit IDs, replace the 6-digit ID with its pairing 2-digit ID.
```

Comment:
would it make sense to add this information as well: [internal room id, unique room id, room type], in addition to your example?